DiscoverOctoPrintAction.qml

```diff
@@ -432,11 +432,23 @@ Cura.MachineAction
                         }
                     }
                     CheckBox
+                                        {
+                        id: autoPSUControlCheckBox
+                        text: catalog.i18nc("@label", "Automatically turn on Printer with PSUControl")
+                        enabled: manager.instanceApiKeyAccepted
+                        checked: manager.instanceApiKeyAccepted && Cura.ContainerManager.getContainerMetaDataEntry(Cura.MachineManager.activeMachineId, "octoprint_psu_control") == "true"
+                        onClicked:
+                        {
+                            manager.setContainerMetaDataEntry(Cura.MachineManager.activeMachineId, "octoprint_psu_control", String(checked))
+                        }
+                    }
+                    
+                     CheckBox
                     {
                         id: showCameraCheckBox
                         text: catalog.i18nc("@label", "Show webcam image")
                         enabled: manager.instanceSupportsCamera
-                        checked: manager.instanceApiKeyAccepted && Cura.ContainerManager.getContainerMetaDataEntry(Cura.MachineManager.activeMachineId, "octoprint_show_camera") != "false"
+                        checked: manager.instanceApiKeyAccepted && Cura.ContainerManager.getContainerMetaDataEntry(Cura.MachineManager.activeMachineId, "octoprint_show_camera") != "true"
```

Comment:
This seems wrong/unintended.

Reply:
this is a oneliner ...looks unintended because of the new line ...it starts at checked: